src/main/kotlin/uk/gov/dluhc/printapi/database/repository/TemporaryCertificateRepository.kt

```diff
@@ -9,5 +9,5 @@ import java.util.UUID
 @Repository
 interface TemporaryCertificateRepository : JpaRepository<TemporaryCertificate, UUID> {
 
-    fun findByGssCodeInAndSourceTypeAndSourceReference(gssCodes: List<String>, sourceType: SourceType, sourceReference: String): TemporaryCertificate?
+    fun findByGssCodeInAndSourceTypeAndSourceReference(gssCodes: List<String>, sourceType: SourceType, sourceReference: String): List<TemporaryCertificate>
```

Comment:
FYI @abdullah-sanver-valtech

Reply:
Thanks, for addressing it @NathanRussellValtech.